docs/content/drive.md

```diff
@@ -582,7 +584,7 @@ Properties:
 
 #### --drive-scope
 
-Scope that rclone should use when requesting access from drive.
+Comma separated list of scopes that rclone should use when requesting access from drive.
```

Comment:
Note that this change should go here in the code

https://github.com/rclone/rclone/blob/29baa5888fc149615cc132f259c29a21a40769dd/backend/drive/drive.go#L255

As this part of the drive docs are auto generated from the code.